xml/System.Security.Cryptography/ECDsaCng.xml

```diff
@@ -134,18 +139,23 @@
       </Parameters>
       <Docs>
         <param name="keySize">The size of the key. Valid key sizes are 256, 384, and 521 bits.</param>
-        <summary>Initializes a new instance of the <see cref="T:System.Security.Cryptography.ECDsaCng" /> class with a random key pair, using the specified key size.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Security.Cryptography.ECDsaCng" /> class with a specified target key size.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The random key pair will have the public key length defined by the `keySize` parameter.  
+
+This constructor does not generate a new public/private keypair immediately.
+This constructor sets the <xref:System.Security.Cryptography.ECDsaCng.KeySize> property to the provided value and when a key is needed the saved size is used to identify the target curve.
+If a key is loaded via the <xref:System.Security.Cryptography.ECDsaCng.ImportParameters%2A> method, or other key import method, the key size from this constructor has no meaning.
+ 
   
  ]]></format>
         </remarks>
         <exception cref="T:System.PlatformNotSupportedException">Cryptography Next Generation (CNG) classes are not supported on this system.</exception>
         <exception cref="T:System.Security.Cryptography.CryptographicException">
           <paramref name="keySize" /> specifies an invalid length.</exception>
+        <altmember cref="M:System.Security.Cryptography.ECDsa.Create(System.Int32)"/>
```

Comment:
The build [had a warning](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C25%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C201911251733433084-3564%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=CFGX0qjAqjq356%2FYCreWkHb53MV0l24SaaGm0YaZBmc%3D&st=2019-11-25T18%3A07%3A32Z&se=2019-12-26T18%3A12%3A32Z&sp=r) stating that this cref does not exist ([here's the preview of this constructor](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.ecdsacng.-ctor?view=netframework-4.8&branch=pr-en-us-3564#System_Security_Cryptography_ECDsaCng__ctor_System_Int32_)).

The ECDsa.xml at the top of this review shows that these are the available DocIds for the Create overloads. Which one did you mean to use?:

`M:System.Security.Cryptography.ECDsa.Create`
`M:System.Security.Cryptography.ECDsa.Create(System.Security.Cryptography.ECCurve)`
`M:System.Security.Cryptography.ECDsa.Create(System.Security.Cryptography.ECParameters)`
`M:System.Security.Cryptography.ECDsa.Create(System.String)`